### DISCUSSION TITLE
How to get individual step execution details

### DISCUSSION THREAD

**Discussion by far11ven** [badeball/cypress-cucumber-preprocessor#905] [General]
previously I had been using **@Klaveness-Digital/cypress-cucumber-preprocessor**

and I could get a test step state like the below:

```
  let scenarioName = window.testState.currentScenario.name;
  let stepResult = window.testState.stepResults;

  window.testState.scenarioSteps[scenarioName].forEach(function(currStep,index){ 
    if(stepResult[index].status.toUpperCase() === "PASSED" ) {
        console.log("currStep execution details", stepResult[index].status)
    }
  }
```
But now ``` window.testState```  doesn't have this detail, is this detail available through some other API? @badeball 

``` window.testState```

![image](https://user-images.githubusercontent.com/26922130/207075172-c33fbaed-59b6-4532-b431-34ecc8df33d8.png)

**Comment by badeball** [comment]
No, it isn't. What were you using it for?

**Reply by far11ven** [reply]
I had been using the above in some custom logic during reporting.

**Reply by SamBerryy** [reply]
@far11ven Have you managed to build a workaround for this? I'm needing something similar
